nuntium/templates/nuntium/message/post_submission.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="jumbotron">
+    <div class="container">
+        <h1>{% trans "Your message has been created!" %}</h1>
+        {% if messages %}
```

Comment:
Are you getting two things mixed up here? This template variable 'messages' is from the Django Messaging Framework (https://docs.djangoproject.com/en/1.7/ref/contrib/messages/), isn't it? So it's just the general purpose code for displaying flash messages across the top of whatever page is being viewed - normally I'd expect this stuff to be in a base template so that it appears everywhere.

Or have I got the wrong end of the stick?